.github/workflows/auto-update-otel-sdk.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           java-version: 17.0.6
 
       - name: Update license report
-        uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3.1.0
+        uses: gradle/actions/setup-gradle@v3
```

Comment:
could you pin the action version using the hash and add the actual version as comment